imports/plugins/core/accounts/server/methods/inviteShopOwner.js

```diff
@@ -57,7 +57,8 @@ export default function inviteShopOwner(options, shopData) {
   const currentUser = Meteor.user();
   const currentUserName = getCurrentUserName(currentUser);
   // uses primaryShop's data (name, address etc) in email copy sent to new merchant
-  const dataForEmail = getDataForEmail({ shop: primaryShop, currentUserName, name, token });
+  const context = Promise.await(getGraphQLContextInMeteorMethod(Reaction.getUserId()));
+  const dataForEmail = getDataForEmail(context, { shop: primaryShop, currentUserName, name, token });
```

Comment:
Wrap the `getDataForEmail` call in `Promise.await()` since that function is async now.